.github/workflows/build.yml

```diff
@@ -25,8 +41,19 @@ jobs:
           distribution: 'temurin'
           cache: gradle
 
-      # Only library assets are needed for the build step
-      - name: Build with Gradle
-        run: ./gradlew :${{ matrix.codebase }}:assemble
+      #After decoding the secret key, place the file in ~ /. Gradle/ secring.gpg
+      - name: Decode Signing Key
+        uses: ./.github/actions/decode_signing_key_action
+        with:
+          signing_key_file: ${{ secrets.SIGNING_KEY_FILE }}
+          signing_file_path: ${{ env.SIGNING_KEY_FILE_PATH }}
+
+      - name: Assemble
+        run: ./gradlew --stacktrace assemble -x :demo:assemble # we exclude Demo module in assemble
```

Comment:
Since we only have the one module, we should be able to just call `./gradlew --stacktrace assemble :library:assemble`

Reply:
updated